sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -15531,7 +15595,7 @@
           "encrypted": false,
           "redrawOn": "data",
           "customDefaultValue": "",
-          "calculateValue": "if (data.hasDependents === 'no' && (data.relationshipStatus === 'single') && data.outOfHighSchoolFor4Years === 'no' && data.howWillYouBeAttendingTheProgram === \"Full Time\") {\n  instance.setValue (true);\n}\nelse {\n  instance.setValue (false);\n}",
+          "calculateValue": "if (\r\n  data.hasDependents === \"no\" &&\r\n  data.relationshipStatus === \"single\" &&\r\n  data.outOfHighSchoolFor4Years === \"no\" &&\r\n  data.howWillYouBeAttendingTheProgram === \"Full Time\" &&\r\n  !(data.everInCustodyOfChildWelfare === \"yes\")\r\n) {\r\n  instance.setValue(true);\r\n} else {\r\n  instance.setValue(false);\r\n}",
```

Comment:
Minor. I see a warning in form editor. 
![image](https://github.com/bcgov/SIMS/assets/54600590/30631aba-1029-4ce3-b373-f6c6d1cc9307)
 
we can use `data.everInCustodyOfChildWelfare !== "yes"` instead